Source/WTF/wtf/text/IntegerToStringConversion.h

```diff
@@ -91,8 +91,8 @@ inline void writeIntegerToBuffer(IntegerType integer, CharacterType* destination
         return writeIntegerToBufferImpl<CharacterType, uint8_t, PositiveNumber>(integer ? 1 : 0, destination);
     else if constexpr (std::is_signed_v<IntegerType>) {
         if (integer < 0)
-            return writeIntegerToBufferImpl<CharacterType, typename std::make_unsigned_t<IntegerType>, NegativeNumber>(-integer, destination);
-        return writeIntegerToBufferImpl<CharacterType, typename std::make_unsigned_t<IntegerType>, PositiveNumber>(integer, destination);
+            return writeIntegerToBufferImpl<CharacterType, typename std::make_unsigned_t<IntegerType>, NegativeNumber>(std::make_unsigned_t<IntegerType>(-integer), destination);
```

Comment:
As far as I can tell, the syntax here is incorrect because `std::make_unsigned_t<IntegerType>` describes a _type_, not a conversion function.

The proper way to do this is to use `static_cast<>()` with the type like this:

```
static_cast<std::make_unsigned_t<IntegerType>>(-integer)
```

All four call sites need to be fixed.